terraform-tests/postgres_test.go

```diff
@@ -35,7 +36,7 @@ var _ = Describe("postgres", Label("postgres-terraform"), Ordered, func() {
 		"rds_subnet_group":                      "",
 		"rds_vpc_security_group_ids":            "",
 		"allow_major_version_upgrade":           true,
-		"auto_minor_version_upgrade":            true,
+		"auto_minor_version_upgrade":            false,
```

Comment:
The default value in the yml file is `true`.  I think we should test true + major version here and then have separate test cases for `false` and `true` + minor version

Reply:
Great idea! I shouldn't change this value